kafka/src/main/java/io/micronaut/configuration/kafka/event/AbstractKafkaApplicationEvent.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.configuration.kafka.event;
+
+import io.micronaut.context.event.ApplicationEvent;
+
+/**
+ * Abstract base class for all kafka application events.
+ *
+ * @author Jorge F. Sanchez
+ * @since 4.5.3
+ * @param <T> the {@link ApplicationEvent} source object type
+ */
+public class AbstractKafkaApplicationEvent<T> extends ApplicationEvent {
+    /**
+     * Constructs a prototypical Event.
+     *
+     * @param source The object on which the Event initially occurred.
```

Comment:
```suggestion
     * Constructs an event with a given Consumer source.
     *
     * @param consumer The Consumer on which the Event initially occurred.
```

Reply:
My mistake, I didn't mean to re-word this one since it is generic and doesn't strictly apply to a Consumer - and now CI is failing because of the `@param` name not matching. I think this one can go back to the way it was.